.github/workflows/CODEOWNERS

```
@@ -0,0 +1,20 @@
+clang/ @erichkeane, @Fznamznon
+
+clang/include/clang/Driver/ @mdtoguchi @AGindinson
+clang/lib/Driver/ @mdtoguchi @AGindinson
+
```

```suggestion
clang/test/Driver/ @mdtoguchi @AGindinson

```
Or maybe, something like this could work for the whole component? (based on https://git-scm.com/docs/gitignore#_pattern_format)
```
clang/**/Driver/ @mdtoguchi @AGindinson 
```

---

Done.